modules/private/hook.go

```diff
@@ -79,6 +81,22 @@ type HookPostReceiveBranchResult struct {
 	URL     string
 }
 
+// HockProcReceiveResult represents an individual result from ProcReceive
+type HockProcReceiveResult struct {
+	Results []HockProcReceiveRefResult
+	Err     string
+}
+
+// HockProcReceiveRefResult represents an individual result from ProcReceive
+type HockProcReceiveRefResult struct {
+	OldOID      string
+	NewOID      string
+	Ref         string
+	OrignRef    string
```

Comment:
I don't like this OrignRef name - This is the original ref that was provided in the proc-receive so either:

```suggestion
	OriginalRef    string
```

or 

```suggestion
	ProvidedRef    string
```

We can probably leave the Ref one as is, but it could be called `AlternateRef`